src/Components/DarkMode/DarkMode.js

```diff
@@ -0,0 +1,13 @@
+import React, { useContext } from 'react';
+import { ThemeContext } from '../../contexts/ThemeContext';
+
+function DarkMode() {
```

Comment:
We might want to rename this to make it a bit more clear that it is a button/toggle

Reply:
When I dive into refactor tonight I can tackle this, if thats ok with everyone.

function DarkModeToggle()